pkg/handler/repository_test.go

```diff
@@ -666,6 +671,13 @@ func (suite *ReposSuite) TestPartialUpdate() {
 	assert.Equal(t, http.StatusOK, code)
 }
 
+//func (suite *ReposSuite) TestValidateParameters() {
+//	t := suite.T()
+//
+//	mockDao := MockRepositoryDao{}
+//	mockDao.On("ValidateParameters", mockOrgId, uuid, expected).Return(nil)
+//}
+
```

Comment:
Did you mean to come back to this?

Reply:
man i'm going senile in my old age

Reply:
ah yes, i wrote the  handler test over in repository_parameter_test, forgot to remove this